x-pack/legacy/plugins/uptime/public/components/functional/filter_group/filter_group.tsx

```diff
@@ -80,23 +94,37 @@ export const FilterGroupComponent = ({
     {
       fieldName: 'url.port',
       id: 'port',
-      isLoading,
-      items: ports,
+      isDisabled: ports.length === 0,
+      loading,
+      items: ports.map((p: number) => p.toString()),
       onFilterFieldChange,
       selectedItems: getSelectedItems('url.port'),
       title: i18n.translate('xpack.uptime.filterBar.options.portLabel', { defaultMessage: 'Port' }),
     },
     {
       fieldName: 'monitor.type',
       id: 'scheme',
-      isLoading,
+      isDisabled: schemes.length === 0,
+      loading,
       items: schemes,
       onFilterFieldChange,
       selectedItems: getSelectedItems('monitor.type'),
       title: i18n.translate('xpack.uptime.filterBar.options.schemeLabel', {
         defaultMessage: 'Scheme',
       }),
     },
+    {
+      fieldName: 'tags',
+      id: 'tags',
+      isDisabled: tags.length === 0,
+      loading,
+      items: tags,
+      onFilterFieldChange,
+      selectedItems: getSelectedItems('tags'),
+      title: i18n.translate('xpack.uptime.filterBar.options.tagsLabel', {
+        defaultMessage: 'Tags',
+      }),
+    },
```

Comment:
Usually it's a good practice to put shorthand properties of the object at the beginning, it's better readable. For example for 'onFilterFieldChange', 'loading' .

Reply:
I agree that is cleaner.